tests/rkt_cat_manifes_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+)
+
+const (
+
+	// The expected image manifest of the 'rkt-inspect-image-cat-manifest.aci'.
+	manifestTemplate = `{"acKind":"ImageManifest","acVersion":"0.5.1","name":"IMG_NAME","labels":[{"name":"version","value":"1.0.0"},{"name":"arch","value":"amd64"},{"name":"os","value":"linux"}],"app":{"exec":["/inspect"],"user":"0","group":"0","workingDirectory":"/","environment":[{"name":"VAR_FROM_MANIFEST","value":"manifest"}]}}`
+)
+
+// TestCatManifest tests 'rkt image cat-manifest', it will:
+// Read some existing image manifest via the image name, and verify the result.
+// Read some existing image manifest via the image hash, and verify the result.
+// Read some non-existing image manifest via the image name, and verify nothing is found.
+// Read some non-existing image manifest via the image hash, and verify nothing is found.
+func TestCatManifest(t *testing.T) {
+	var testImageHash string
+	testImage := "rkt-inspect-image-cat-manifest.aci"
+	testImageName := "coreos.com/rkt-cat-manifest-test"
+	expectManifest := strings.Replace(manifestTemplate, "IMG_NAME", testImageName, -1)
+
+	tmpManifest, err := ioutil.TempFile("", "")
```

Comment:
I would like to have a prefix for the temporary files, so if the test is killed, it is easier to understand what is in `/tmp`. Something like:
```
	tmpManifest, err := ioutil.TempFile("", "rkt-TestCatManifest-")
```